QUEST_LV_0100.tsv

```diff
@@ -638,21 +638,21 @@ QUEST_LV_0100_20150317_000637	$Venucelos are very brave and loyal.{nl}But, if th
 QUEST_LV_0100_20150317_000638	$Zachariel was proud to make huge Guardians.{nl}He even put secret treasures inside them so that only the one who destroy these Guardians could get them.
 QUEST_LV_0100_20150317_000639	$The forces of evil will never reach the Goddess' revelation.
 QUEST_LV_0100_20150317_000640	Coben
-QUEST_LV_0100_20150317_000641	I am looking for my brother.{nl}If you could help me, I may give you some rewards.
-QUEST_LV_0100_20150317_000642	He was really fanstastic.{nl}A person like me can not imagine.
-QUEST_LV_0100_20150317_000643	Something is between the pages of diary.{nl}Blueprint.. Brother got something again.
-QUEST_LV_0100_20150317_000644	I came here to look for my brother, but I am not as brave as him.{nl}I want you to take a look at the gravestone's writing on my behalf.
-QUEST_LV_0100_20150317_000645	Yes. My brother is a thief.{nl}My brother used to come to our shop once in a few years and told us his adventures.
-QUEST_LV_0100_20150317_000646	So he challenged for the Great Mage Tower?{nl}That's my brother! He can certainly do that.{nl}
-QUEST_LV_0100_20150317_000647	The Mage Tower was the fortress that the demons could not come near.
-QUEST_LV_0100_20150317_000648	We should look for the next gravestone.{nl}If the writing can't be seen clearly, rub it with Cockatrice fat.
-QUEST_LV_0100_20150317_000649	Mage Tower.. Do we have anything to steal there?
-QUEST_LV_0100_20150317_000650	Yes! He will not end after stealing at the Mage Tower.{nl}I will go to Negybas Field first so follow me!
-QUEST_LV_0100_20150317_000651	You came. The next gravestone is in front of me, but I am just a drunken coward.{nl}I will stay at the road on the left side of Negybas Field so you go and check it.
-QUEST_LV_0100_20150317_000652	It is a shame to look back once a man starts his journey.{nl}My brother will rather choose to die than looking back.
-QUEST_LV_0100_20150317_000653	How could he think of using magic to the great magician?{nl}That's my brother.{nl}
-QUEST_LV_0100_20150317_000654	{memo X}We should check by rubbing Hallowventer's wooden coals.{nl}The next epitaph is at the right side of Negibas Fields.
-QUEST_LV_0100_20150317_000655	He will do anything if he decides to do something.{nl}How could we have imagined that he will climb onto Mage Tower which even demons failed to climb?
+QUEST_LV_0100_20150317_000641	$I am looking for my brother.{nl}If you can help me, I may give you some rewards.
+QUEST_LV_0100_20150317_000642	$He was a really fantastic person.{nl}A dream guy for people like me.
```

Comment:
A dream guy for people like me. Sounds awkward.
Suggestion: I can't imagine myself being like him.